fruit_classifier/train/train_utils.py

```diff
@@ -304,6 +305,7 @@ def plot_training(history):
     plt.xlabel('Epoch #')
     plt.ylabel('Loss/Accuracy')
     plt.legend(loc='lower left')
+    plt.show()
```

Comment:
`plt.show()` can break the rendering, and there may be places where a `DISPLAY` variable is empty.
There are two solutions to this:

1. Don't show the plot
2. Return the `Axes` object if the user explicitly wants to plot